Doc/library/functions.rst

```diff
@@ -763,6 +763,9 @@ are always available.  They are listed here in alphabetical order.
    .. versionchanged:: 3.6
       Grouping digits with underscores as in code literals is allowed.
 
+   .. versionchanged:: 3.7
+      The first parameter can no longer be passed by keyword.
```

Comment:
As a non-native English speaker, I think I could understand "be passed as a keyword argument" better than "passed by keyword". Or "the first parameter can only be passed as a positional argument".

Reply:
I'm not a native speaker either so I wasn't sure how to properly word it. "The first parameter can no longer be passed as a keyword argument" seems easiest to understand indeed.